tests/mock_network/mod.rs

```diff
@@ -39,57 +39,12 @@ fn test_nodes(percentage_size: usize) {
     let size = LOWERED_ELDER_SIZE * percentage_size / 100;
     let env = Environment::new(NetworkParams {
         elder_size: LOWERED_ELDER_SIZE,
-        safe_section_size: LOWERED_ELDER_SIZE,
+        safe_section_size: LOWERED_ELDER_SIZE + 1,
```

Comment:
Why is it 5 now instead of 3?

Reply:
So `elder_size` changed from 3 to 4 because 4 is the smallest number where we can still lose 1 elder and still be able to reach consensus on it (because 3 is more than 2/3 of 4) which is what we need to do now (to trigger splits). I changed `safe_section_size` to be one more than `elder_size` just so it's not the same value. It would probably still work, but would be less realistic.

Reply:
OK. I think these things are good to have as comments in there when some magic number is used/updated. It becomes very obscure for someone looking at it otherwise.